docs/source/defer-support.md

```diff
@@ -77,6 +75,25 @@ One drawback of using Multipart HTTP is that there is generally a finite browser
 
 We are working on refactoring the request pipeline in Apollo Server to make it easier to add support for other transport modules.
 
+## Apollo Server Variants
+In order to support `@defer`, Apollo Server variants like Koa, Hapi etc must explicitly support and enable it. This is done by passing in an `enableDefer` flag to `runHttpQuery`. For illustration, this is how it looks like on `apollo-server-express`. Without this flag, the `@defer` directive will be ignored. 
+
+```typescript
```

Comment:
Please convert this to a JavaScript example. We no longer use TypeScript examples in the code samples for docs to enable easy copy-and-paste interaction from developers

Reply:
This code is not client facing, but rather for maintainers of other apollo server variants.

Reply:
Oh Okay. That makes sense!